src/sentry/coreapi.py

```diff
@@ -57,15 +51,6 @@ class APIForbidden(APIError):
     http_status = 403
 
 
-class APIRateLimited(APIError):
-    http_status = 429
-    msg = "Creation of this event was denied due to rate limiting"
-    name = "rate_limit"
-
-    def __init__(self, retry_after=None):
-        self.retry_after = retry_after
-
-
```

Comment:
`ClientAuthHelper` and most of what it uses internally is completely dead now, other than in tests. Let's remove that, too.